.circleci/config.yml

```diff
@@ -9,6 +9,7 @@ jobs:
     branches:
       only:
         - gh-pages
+        - /.*/
```

Comment:
Hi, @Naylin15 
In our documentation we explain the way to create features branches?
Because normally the user can select to use "-" or  "/"
So, we need to explain that we support only "/"·

Reply:
Yes, we do, we give examples and emphasize the branch format

https://git.io/fNBmH